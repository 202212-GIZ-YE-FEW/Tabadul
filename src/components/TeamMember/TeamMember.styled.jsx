import styled from "@emotion/styled";
import Image from "next/image";

export const TeamCard = styled.div`
    margin: auto;
    @media screen and (max-width: 768px) {
        display: flex;
        justify-content: center;
        flex-direction: column;
        align-items: center;
    }
`;
export const TeamImg = styled(Image)`
    max-width: 100%;
    border-radius: 50%;
    object-fit: contain;
    @media (min-width: 1200px) and (max-width: 1500px) {
        width: 250px;
    }
    @media (min-width: 768px) and (max-width: 1200px) {
        width: fit-content;
    }
    @media screen and (max-width: 768px) {
        width: 118px;
        height: 118px;
    }
`;
export const TeamTitle = styled.h3`
    font-style: normal;
    font-weight: 700;
    font-size: 24px;
    line-height: 20px;
    text-align: center;
    color: #000000;
    @media (min-width: 1200px) and (max-width: 1500px) {
        font-size: 20px;
    }
    @media (min-width: 992px) and (max-width: 1200px) {
        font-size: 18px;
    }
    @media screen and (max-width: 992px) {
        font-size: 20px;
        line-height: 20px;
        text-align: center;
    }
`;
export const TeamDesc = styled.p`
    font-style: normal;
    font-weight: 400;
    font-size: 20px;
    line-height: 29px;
    text-align: center;
    width: 200px;
    margin: auto;
    @media (min-width: 1200px) and (max-width: 1500px) {
        font-size: 18px;
    }
    @media (min-width: 992px) and (max-width: 1200px) {
        font-size: 18px;
    }
    @media screen and (max-width: 992px) {
        font-weight: 400;
        font-size: 20px;
        line-height: 29px;
        text-align: center;
    }
`;
