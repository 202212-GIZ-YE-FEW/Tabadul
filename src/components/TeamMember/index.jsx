import React from "react";

import { TeamCard, TeamDesc, TeamImg, TeamTitle } from "./TeamMember.styled";
function TeamMember({ surename, desc, Imgmember }) {
    return (
        <TeamCard>
            <TeamImg
                src={Imgmember}
                alt='member'
                width={300}
                height={300}
                property='true'
            />
            <TeamTitle>{surename}</TeamTitle>
            <TeamDesc>{desc}</TeamDesc>
        </TeamCard>
    );
}

export default TeamMember;
