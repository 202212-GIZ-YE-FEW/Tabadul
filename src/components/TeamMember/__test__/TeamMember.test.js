import renderer from "react-test-renderer";

import TeamMember from "..";

it("renders correctly", () => {
    const tree = renderer.create(<TeamMember />).toJSON();
    expect(tree).toMatchSnapshot();
});
