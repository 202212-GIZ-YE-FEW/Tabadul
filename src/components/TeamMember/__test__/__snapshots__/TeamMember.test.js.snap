// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-wr9924"
>
  <img
    alt="member"
    className="css-1v1g91h"
    data-nimg="1"
    decoding="async"
    height={300}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    property="true"
    src=""
    style={
      Object {
        "color": "transparent",
      }
    }
    width={300}
  />
  <h3
    className="css-1e03y3q"
  />
  <p
    className="css-1hpa51y"
  />
</div>
`;
